docs/configuration/reference/rules.md

```diff
@@ -38,6 +38,14 @@ The `rules` block can be used at the root of a configuration file, or inside an
 
 ---
 
+- message
+- string
+- Optional custom message for this rule.
+  Example: `My Error Derscription. {{message}}`.
```

Comment:
```suggestion
  Example: `My Error Description. {{message}}`.
```

Reply:
I'm not sure I understand. Is the custom message what goes before {{message}} ? Or do users put the message in the brackets in place of "message"?

Reply:
>  Is the custom message what goes before {{message}} ?

Yes. The `{{message}}` placeholder is to be replaced by the linter. BTW, users could just put something like `My Custom Message` there without using the `{{message}}` at all. Then, they won't be able to see the default (or the "actual") linter message.